.github/pull_request_template.md

```diff
@@ -1,26 +1,33 @@
-## Description
-
-<!-- A brief description of the change being made with this pull request. -->
-
 ## Motivation
 
 <!-- What inspired you to submit this pull request? -->
 
+## Changes
+
+<!-- A brief description of the change being made with this pull request. -->
+
 ## Workflow
 
-1. ⚠️⚠️ Create your PR as draft
+
+1. ⚠️ Create your PR as draft ⚠️
 2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
 3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
 4. Mark it as ready for review
+    * Test logic is modified? -> Get a review from RFC owner
+    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
+
+:rocket: Once your PR is reviewed, you can merge it!
 
-Once your PR is reviewed, you can merge it! :heart:
+🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟
 
 ## Reviewer checklist
 
-* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
-* [ ] CI is green
-   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
-* [ ] if any of `build-some-image` label is present
-  1. is the image labl have been updated ? 
-  2. just before merging, locally build and push the image to hub.docker.com
-* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly
+* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
```

Comment:
I assume this labelling could be done automatically, worst case you fallback on run-all-scenarios.
We have a labeller in several repos already if a commonly available one wouldn't match this use case.

Reply:
Well, it's actually not easy, because it's not based on filenames.

I'm really scared of automatons that almost works: it removes the responsability of checking, leading to catastrophic failure every time the solution does not work. I even prefer solution that works 50% : people knows it's crap, and thus are very careful 😄.